src/tests/commonTests.ts

```diff
@@ -85,12 +90,14 @@ export async function renders(
     display: string;
   }
 ): Promise<void> {
-  const page = await simplePageSetup(componentTagOrHTML);
-  const element = await page.find(getTag(componentTagOrHTML));
+  it("renders", async () => {
```

Comment:
should it maybe be...

```ts
it(`renders: ${componentTagOrHTML}`, async () => {
```

Just thinking it would be nice if the "renders" test was named unique to the HTML/tag